GitCommands/Settings/AppSettings.cs

```diff
@@ -876,7 +876,7 @@ public static bool ShowGitNotes
 
         public static bool ShowIndicatorForMultilineMessage
         {
-            get => GetBool("showindicatorformultilinemessage", false);
+            get => GetBool("showindicatorformultilinemessage", true);
```

Comment:
This setting has performance implications, slower loading. OK for me, I use the setting.)

Reply:
Oh thanks, I wasn't particularly aware of this.
Perhaps we want to keep it disabled - our work laptops are crippled with McAfee and I wouldn't want to make engineers even less efficient than they are already...

Reply:
Reverting my comment: drew fixed this in #4988, this is no longer limiting performance.